change/@microsoft-teams-js-24933635-de42-4b04-b53f-935c1799929a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
This shouldn't be a major change, major indicates that this is a breaking change for developers that will require them to make a code change on their side. I would this this is a patch since it is not changing developer facing functionality and won't require them to "do" anything.

Reply:
Thank you for letting me know. I will change it in next push. Do we have any documentation or somewhere defined what each term means (major, minor, patch)? I am just curious about what kind of change is defined as minor.